app-i18n/man-pages-it/man-pages-it-2.80-r1.ebuild

```diff
@@ -1,10 +1,10 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
```

Comment:
Please don't change EAPI in stable ebuilds. It could break stuff for stable users…

…also please test your changes because it obviously does.